@plotly/dash-generator-test-component-typescript/src/components/RequiredChildrenComponent.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { RequiredChildrenComponentProps } from "../props";
+
+
+const RequiredChildrenComponent = (props: RequiredChildrenComponentProps) => {
+  const {children} = props;
+  return <div>
+    {children}
+  </div>
```

Comment:
I think it would be good to wrap it in parentheses.

Reply:
Thanks! It looks better 😄